misc/slack-notification/on-start.sh

```diff
@@ -6,7 +6,13 @@ else
   ACTION="Continued"
 fi
 
-MESSAGE="${ACTION} running \`${ENV0_DEPLOYMENT_TYPE}\` deployment."
+if [ "${ENV0_DEPLOYER_NAME}" = "env0" ]; then
+  TRIGGER="Scheduled"
```

Comment:
idk about scheduled since also CD will have env0 as the deployed
maybe system?

Reply:
f30265c4